docs/user_guide/HOWTO/Upgrading_SIMP.rst

```diff
@@ -6,7 +6,7 @@ HOWTO Upgrade SIMP
 .. WARNING::
```

Comment:
This whole file should either be left alone or deleted for now. It will need to be heavily rewritten for SIMP 6 anyway.